src/parser/paladin/holy/modules/Abilities.tsx

```diff
@@ -184,7 +184,7 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.AVENGING_WRATH,
         category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-        cooldown: 120,
+        cooldown: (this.selectedCombatant.hasMajor(SPELLS.VISION_OF_PERFECTION.traitId)) ? 120 * 0.79 : 120,
```

Comment:
How about AC?

Reply:
Yes, both flavors of wings cooldowns are effected.  I have updated them accordingly.